src/app/components/mnoe-users-local-list/mnoe-users-local-list.coffee

```diff
@@ -78,6 +78,66 @@
           MnoErrorsHandler.processServerError(error)
       ).finally(-> user.isSendingInvite = false)
 
+    scope.availableRoles = () ->
```

Comment:
why not use the constant instead of creating a function?
check out `index.constants.coffee`